astropy/cosmology/funcs/optimize.py

```diff
@@ -152,7 +152,7 @@ def z_at_value(
     func : function or method
         A function that takes a redshift as input.
 
-    fval : `~astropy.units.Quantity`
+    fval : Quantity
```

Comment:
`['redshift']`?

Reply:
it can be any dimension, but is probably distance.